async-nats/src/lib.rs

```diff
@@ -1251,6 +1293,48 @@ impl Subscriber {
             .await?;
         Ok(())
     }
+
+    /// Unsubscribes immediately but leaves the stream open to allow any in-flight messages on the
```

Comment:
Maybe `subscription` instead of `stream`?  Sounds more direct on what is affected.